<launch>
    <node pkg="rclcpp_components" exec="component_container" name="odometry_bridge">
    
    </node>
    <load_composable_node target="OdometryBridge">
        <composable_node pkg="odometry_plugins" plugin="odometry_bridge::OdometryBridge" name="odometry_bridge">
        </composable_node>
    </load_composable_node>
</launch>